DeepSpeech.py

```diff
@@ -834,6 +843,7 @@ def do_graph_freeze(output_file=None, output_node_names=None, variables_blacklis
         except RuntimeError as e:
             log_error(str(e))
 
+
```

Comment:
This was on purpose to match PEP 8 (two empty lines between top-level functions)